MatterControlLib/SlicerConfiguration/Settings/ProfileManager.cs

```diff
@@ -59,7 +57,6 @@ public class ProfileManager
 
 		static ProfileManager()
 		{
-			PrinterSettings.SettingChanged.RegisterEvent(SettingsChanged, ref unregisterEvents);
```

Comment:
@larsbrubaker - what was the plan to replace this? We previously listened for SettingsChanged and updated profile name and comport metadata. It looks like we're no longer doing that as the local SettingChanged event listener appears to be unreferenced. I assume we want to register the SettingsChanged listener to the AnySettingChanged event that may or may not have been created